src/mobu/handlers/github_webhooks.py

```diff
@@ -0,0 +1,42 @@
+"""Github webhook handlers."""
+
+from gidgethub import routing
+from gidgethub.sansio import Event
+from structlog.stdlib import BoundLogger
+
+from ..dependencies.context import RequestContext
+
+__all__ = ["webhook_router"]
+
+webhook_router = routing.Router()
+
+
+@webhook_router.register("push")
+async def handle_push(
+    event: Event, logger: BoundLogger, context: RequestContext
+) -> None:
+    """Handle a push event."""
+    ref = event.data["ref"]
+    url = event.data["repository"]["clone_url"]
```

Comment:
I think you can use `safir.github.webhooks.GitHubPushEventModel` rather than manually traversing the raw dictionary. That way, we only have to change one place if GitHub ever changes their models.

Reply:
That model assumes we'll have an attribute related to github applications that we don't have here, I don't remember offhand what it is.

Reply:
Ah, okay, makes sense! I don't really understand the different types of GitHub integrations.